scripts/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# This assumes you have general prerequisites installed as by:
+# https://github.com/artsy/potential/blob/master/scripts/setup
+
+# Exit if any subcommand fails
+set -e
+
+if [ ! -z $NVM_DIR ]; then # skip nvm steps if not available
+  echo "Configuring node 12"
+  source ~/.nvm/nvm.sh
+  nvm install
+fi
+
+echo 'Install yarn and memchached'
+brew bundle
+
+echo 'Using yarn to install required node packages'
+yarn install
+yarn add dotenv-cli
```

Comment:
I think this command will _update_ the `package.json` file and result in changes that must be committed. If it's necessary, I believe the right thing to do is to add it to the project's `devDependencies` so the `yarn install` command above will take care of installing it along with other packages.